co-simulation/fed/mosaic-carma/pom.xml

```diff
@@ -54,6 +59,12 @@
       <version>4.11</version>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.eclipse.mosaic</groupId>
+          <artifactId>mosaic-application</artifactId>
+          <version>22.1-SNAPSHOT</version>
+          <scope>compile</scope>
+      </dependency>
   </dependencies>
```

Comment:
Sorry I was missing this file from my last review. 
Since CarmaMessageAmbassador.java imports javax.xml.bind.DatatypeConverter, it should be added a dependency like:

    <dependency>
      <groupId>javax.xml.bind</groupId>
      <artifactId>jaxb-api</artifactId>
      <version>2.3.1</version>
    </dependency>